lib/UsageRecord.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Stripe;
+
+/**
+ * Class UsageRecord
+ *
+ * @package Stripe
+ *
+ * @property string $id
+ * @property string $object
+ * @property bool $livemode
+ * @property int $quantity
+ * @property string $subscription_item
+ * @property int $timestamp
+ */
+class UsageRecord extends ApiResource
+{
+    /**
+     * @param array|null $params
+     * @param array|string|null $options
+     *
+     * @return \Stripe\ApiResource The created resource.
+     */
+    public static function create($params = null, $options = null)
+    {
+        self::_validateParams($params);
+        $subscription_item = $params['subscription_item'];
```

Comment:
Maybe do the same thing that you did in stripe-java? I.e. add a check and throw `\Stripe\Error\InvalidRequest` exception if the parameter is missing.